### PR TITLE
feat(metrics): add 2fa setup successful code submission event

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

File: packages/fxa-settings/src/pages/InlineTotpSetup/container.tsx [hunk 1]
```diff
@@ -146,6 +147,7 @@ export const InlineTotpSetupContainer = ({
           ...Object.assign({}, signinState),
           ...(totp ? { totp } : {}),
         };
+        GleanMetrics.accountPref.twoStepAuthQrCodeSuccess();
```
Comment (chenba) [inline packages/fxa-settings/src/pages/InlineTotpSetup/container.tsx:150]: During MozWeek, Valerie and I chatted with Ross, and he's okay with the two 2fa setups (settings and inline) use the same events.

**Review by vpomerleau — APPROVED** [review]
Tested locally and see the event coming through! 🙌

File: packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml [hunk 2]
```diff
@@ -1966,6 +1966,23 @@ account_pref:
...
+    type: event
+    description: |
+      Authentication code was successfully entered and submitted.
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9573
```
Comment (vpomerleau) [inline packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml:1979]: Should this be 9576?

Comment (chenba) [reply packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml:1979]: Yes, thank you!

**Pull request merged**